src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/AnalyzeContext.cs

```diff
@@ -11,11 +11,8 @@ public class AnalyzeContext
         public AnalyzeContext(IUpgradeContext context)
```

Comment:
Do you even need this context still?

Reply:
I'm pulling project details from UpgradeContext, leaving the AnalyzeContext for now as is.